Makefile

```diff
@@ -64,6 +67,19 @@ coverage: $(ACTIVATE)
 	@echo "Computing unit test coverage..."
 	@. $(ACTIVATE); coverage run --source=ie_serving -m pytest $(TEST_DIRS)/unit/ && coverage report --fail-under=70
 
+ams_image: $(AMS_EXAMPLE)requirements.txt $(AMS_EXAMPLE)requirements-dev.txt
+	echo "Running ams image preprocessing tests" 
+	test -d $(AMS_VIRTUALENV_DIR) || $(VIRTUALENV_EXE) $(AMS_VIRTUALENV_DIR)
+	@. $(AMS_ACTIVATE); pip install -qq -r $(AMS_EXAMPLE)requirements.txt
+	@. $(AMS_ACTIVATE); pip install -qq -r $(AMS_EXAMPLE)requirements-dev.txt
+	@. $(AMS_ACTIVATE); test -d $(AMS_EXAMPLE)tests/test_images || (sh $(AMS_EXAMPLE)tests/get_test_images.sh && mv test_images $(AMS_EXAMPLE)tests/)
+	@. $(AMS_ACTIVATE); pytest  $(AMS_EXAMPLE)tests/test_preprocess_binary_images.py
```

Comment:
I think we could change this line to:
```
@. $(AMS_ACTIVATE); pytest  $(AMS_EXAMPLE)tests/
```
so if new tests will be added, they will be automatically started with this makefile target